lib/spack/docs/build_systems/inteloneapipackage.rst

```diff
@@ -0,0 +1,265 @@
+.. Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+   Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+   SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+.. _inteloneapipackage:
+
+
+====================
+ IntelOneapiPackage
+====================
+
+
+.. contents::
+
+
+oneAPI packages in Spack
```

Comment:
Typo: Should this be capital "O" like "OneAPI"?

Reply:
Decided by marketing. Even when it begins a sentence.

https://software.intel.com/content/www/us/en/develop/tools/oneapi.html